lib/galaxy/tool_util/deps/conda_util.py

```diff
@@ -431,7 +431,7 @@ def install_conda(conda_context, force_conda_build=False):
     with tempfile.NamedTemporaryFile(suffix=".sh", prefix="conda_install", delete=False) as temp:
         script_path = temp.name
     download_cmd = commands.download_command(conda_link(), to=script_path)
-    install_cmd = ['bash', script_path, '-b', '-p', conda_context.conda_prefix]
+    install_cmd = ['bash', script_path, '-b', '-p', shlex.quote(conda_context.conda_prefix)]
```

Comment:
I'm quite sure this is not necessary, and actually breaks it if `conda_context.conda_prefix` contains a space. E.g.:

```
In [1]: import shlex                                                                                                                                                                                       

In [2]: from galaxy.util.commands import shell                                                                                                                                                             

In [3]: shell(["ls", "/tmp/galaxy root"])                                                                                                                                                                  
foo
Out[3]: 0

In [4]: shell(["ls", shlex.quote("/tmp/galaxy root")])                                                                                                                                                     
ls: cannot access "'/tmp/galaxy root'": No such file or directory
Out[4]: 2
```